frontend/app/templates/components/footer-main.hbs

```diff
@@ -0,0 +1,47 @@
+<div class="ui center aligned container">
+  <div class="ui stackable inverted divided grid">
+    <div class="three wide column">
+      <div class="ui inverted link list">
+        <strong class="item">Use Badgeyay</strong>
+        <a class="item" href="#" target="_blank">Sitemap</a>
+      </div>
+    </div>
+    <div class="three wide column">
+      <div class="ui inverted link list">
+        <strong class="item">Categories</strong>
+      </div>
+    </div>
+    <div class="three wide column">
+      <div class="ui inverted link list">
+        <strong class="item">Locations</strong>
+        <a class="item" href="#">India</a>
+        <a class="item" href="#">Singapore</a>
+        <a class="item" href="#">Berlin</a>
+        <a class="item" href="#">Mexico</a>
+        <a class="item" href="#">New York</a>
+        <a class="item" href="#">Hong Kong</a>
+      </div>
+    </div>
+    <div class="three wide column">
+      <div class="ui inverted link list">
+        <strong class="item">Nearby Events</strong>
+      </div>
+    </div>
+    <div class="three wide column">
+      <div class="ui inverted link list">
+        <strong class="item">Follow us on</strong>
+        <a class="item" href="#" target="_blank">Facebook</a>
```

Comment:
Please add links to the social handle platforms

Reply:
There is no Facebook Page of Badgeyay for your Kind Information. I have just added to further update when we have stable app.

Reply:
@ParthS007 You are right, at least you can add `twitter` and `GitHub` links

Reply:
we will update when we have stable app. :smile:

Reply:
@ParthS007 Why should we wait for the future, We can update it now. Just a PR update is needed. Views @djmgit

Reply:
Updated @vaibhavsingh97  :smile: